cli/args/flags.rs

```diff
@@ -3384,6 +3389,14 @@ fn cached_only_arg() -> Arg {
     .help("Require that remote dependencies are already cached")
 }
 
+fn frozen_lockfile_arg() -> Arg {
+  Arg::new("frozen")
+    .long("frozen")
+    .alias("frozen-lockfile")
+    .action(ArgAction::SetTrue)
+    .help("Error out if lockfile is out of date")
+}
```

Comment:
So does this allow for `--frozen=false`? Can you update some unit tests in this file to exercise it?

Reply:
Added